src/Aspire.ProjectTemplates/templates/aspire-empty/9.0/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -92,8 +92,7 @@ private static TBuilder AddOpenTelemetryExporters<TBuilder>(this TBuilder builde
     public static TBuilder AddDefaultHealthChecks<TBuilder>(this TBuilder builder) where TBuilder : IHostApplicationBuilder
     {
         builder.Services.AddHealthChecks()
-            // Add a default liveness check to ensure app is responsive
-            .AddCheck("self", () => HealthCheckResult.Healthy(), ["live"]);
+            .AddApplicationStatus();
```

Comment:
I think we need to keep the `"live"` tag so that the health endpoint added at */alive* works as intended.

Reply:
Got it, keeping `live` tag for now.